prow/apis/prowjobs/v1/types.go

```diff
@@ -126,6 +126,7 @@ const (
 // +kubebuilder:printcolumn:name="StartTime",type=date,JSONPath=`.status.startTime`,description="When the job started running."
 // +kubebuilder:printcolumn:name="CompletionTime",type=date,JSONPath=`.status.completionTime`,description="When the job finished running."
 // +kubebuilder:printcolumn:name="State",type=string,JSONPath=`.status.state`,description="The state of the job."
+// +kubebuilder:metadata:annotations="api-approved.kubernetes.io=https://github.com/kubernetes/test-infra/pull/8669"
```

Comment:
Stay complaint with https://github.com/kubernetes/enhancements/pull/1111. The [previous](https://raw.githubusercontent.com/kubernetes/test-infra/dff7eb06bb4b0d65d2000e2a2cb4fb51dd48d4a3/config/prow/cluster/prowjob-crd/prowjob_customresourcedefinition.yaml) CRD had it too.